lib/sles4sap_publiccloud_basetest.pm

```diff
@@ -140,9 +140,9 @@ sub set_cli_ssh_opts {
       unless $self->{my_instance};
     $ssh_opts //= join(' ',
         '-E', '/var/tmp/ssh_sut.log',
-        '-F', 'none',
+        #'-F', 'none',
         '-o', 'LogLevel=DEBUG3',
-        '-o', 'PasswordAuthentication=no',
+        #'-o', 'PasswordAuthentication=no',
```

Comment:
I do not understand why exactly but only these two are commented. Why not all of them or none? Is this PR specifically about `PasswordAuthentication` . `-F` is
```
       -F configfile
               Specifies  an  alternative  per-user  configuration file.  If a configuration file is given on the command line, the system-wide configuration file (/etc/ssh/ssh_config) will be ignored.  The default for the per-user configuration file is ~/.ssh/config.  If set to “none”, no  configuration files will be read.
```

Is the idea here to  force SAP tests to to use a ssh config file? Why is that? What we miss not doing that?

Reply:
You use Ansible. Ansible does care about the SSH config so without ssh config it's hard to make Ansible to f.e. not validate the ssh host key.